easybuild/easyconfigs/g/GTDB-Tk/GTDB-Tk-2.4.0-foss-2023a.eb

```diff
@@ -0,0 +1,46 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonBundle'
+
+name = 'GTDB-Tk'
+version = '2.4.0'
+
+homepage = 'https://github.com/Ecogenomics/GTDBTk'
+description = "A toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('DendroPy', '4.6.1'),
+    ('matplotlib', '3.7.2'),
+    ('prodigal', '2.6.3'),
+    ('HMMER', '3.4'),
+    ('pplacer', '1.1.alpha19', '', SYSTEM),
+    ('FastANI', '1.34'),
+    ('FastTree', '2.1.11'),
+    ('Mash', '2.3'),
+    ('tqdm', '4.66.1'),
+    ('pydantic', '1.10.13'),
```

Comment:
@asalois Thanks for the contribtuion! The CI is failing, because the default pydantic version for the `foss/2023a` toolchain is 2.5.3. There's an exception in the CI for GTDB-Tk 2.3.2, because it cannot use that version and really needs an older version of pydantic:
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/test/easyconfigs/easyconfigs.py#L614

Based on https://github.com/Ecogenomics/GTDBTk/blob/2.4.0/setup.py#L48, version 2.4.0 still needs an older version of pydantic, so could you add an exception for this version of GTDB-Tk as well in this PR by adding something like the following to that file?
```
            'pydantic': [
                # GTDB-Tk v2.3.2 requires pydantic 1.x (see https://github.com/Ecogenomics/GTDBTk/pull/530)
                ('1.10.13;', ['GTDB-Tk-2.3.2-', 'GTDB-Tk-2.4.0-']),
            ],
```

Reply:
Did I change the correct file?

Reply:
Yes, definitely, the CI is happy now.